src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -209,17 +211,8 @@ EclipseGrid::EclipseGrid(const Deck& deck, const int * actnum)
       m_multzMode(PinchMode::ModeEnum::TOP),
       m_pinchGapMode(PinchMode::ModeEnum::GAP)
 {
-    OpmLog::info("Creating grid");
     if (deck.hasKeyword("GDFILE")){
 
-        if (deck.hasKeyword("COORD")){
-            throw std::invalid_argument("COORD can't be used together with GDFILE");
```

Comment:
Is this not true? Why the change?

Reply:
See the very good commit message ;-)

Reply:
I see. Good to merge when green.

Reply:
Seriously - the new functionality provided here: #1996 ensures that the situation can not arise at all.